torchvision/transforms/transforms.py

```diff
@@ -1630,6 +1630,14 @@ def forward(self, img):
             return F.erase(img, x, y, h, w, v, self.inplace)
         return img
 
+    def __repr__(self):
```

Comment:
Can you add an entry in https://github.com/pytorch/vision/blob/8a820172393b1a1c06411f359dba7a305fd9a9b7/test/test_transforms.py#L1992 that checks that `__repr__` works?
Similar to https://github.com/pytorch/vision/blob/8a820172393b1a1c06411f359dba7a305fd9a9b7/test/test_transforms.py#L498-L499